go.mod

```diff
@@ -1,40 +1,56 @@
 module github.com/Thiht/smocker
 
-go 1.15
+go 1.17
 
 require (
 	github.com/Masterminds/sprig/v3 v3.2.2
+	github.com/facebookgo/grace v0.0.0-20180706040059-75cf19382434
+	github.com/labstack/echo v3.3.10+incompatible
+	github.com/layeh/gopher-json v0.0.0-20201124131017-552bb3c4c3bf
+	github.com/namsral/flag v1.7.4-pre
+	github.com/sirupsen/logrus v1.8.1
+	github.com/smartystreets/assertions v1.2.1
+	github.com/stretchr/objx v0.3.0
+	github.com/teris-io/shortid v0.0.0-20201117134242-e59966efd125
+	github.com/yuin/gluamapper v0.0.0-20150323120927-d836955830e7
+	github.com/yuin/gopher-lua v0.0.0-20210529063254-f4c35e4016d9
+	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
+	layeh.com/gopher-luar v1.0.10
+)
+
+replace (
```

Comment:
Replaces is not the correct way to fix theses in my opinion. It's a fragile module feature that should only be used in development, not to patch things.

For jwt-go, the proper fix is instead to upgrade labstack/echo to v4, which fixed the issue by getting rid of jwt-go in favour of another lib.
Since golang.org/x/crypto is also required by echo, switching to v4 will probably fix the issue too.